test/system/src/test/java/io/pravega/test/system/AbstractFailoverTests.java

```diff
@@ -183,6 +189,80 @@ private void closeWriter(EventStreamWriter<Long> writer) {
         }
     }
 
+
+    CompletableFuture<Void> startWritingIntoTxn(final EventStreamWriter<Long> writer) {
+        return CompletableFuture.runAsync(() -> {
+            while (!testState.stopWriteFlag.get()) {
+                Transaction<Long> txnDebugReference = null;
+                AtomicBoolean txnIsDone = new AtomicBoolean(false);
+
+                try {
+                    Transaction<Long> transaction = writer.beginTxn(5000, 3600000, 29000);
+                    txnDebugReference = transaction;
+
+                    // Sets a recurrent delayed task to ping the txn. It exits when the
```

Comment:
ok, then we can retain it until the PR is merged,  and remove it later, to ensure the system tests pass. Or `ping` can be removed as a part of your PR.

Reply:
I had to add pinging to at least one test case because it was failing due to txns timing out and aborting. I suggest we leave the pinging and remove once we make the change. We can add a change to the test case to the existing issue or create a new one.